sklearn/decomposition/dict_learning.py

```diff
@@ -282,9 +282,9 @@ def sparse_encode(X, dictionary, gram=None, cov=None, algorithm='lasso_lars',
                               check_input=False,
                               verbose=verbose)
         # This ensure that dimensionality of code is always 2,
-        # consistant with the case n_jobs > 1
+        # consistent with the case n_jobs > 1
         if code.ndim == 1:
-            code = code[np.newaxis, :]
+            code = code[:, np.newaxis]
```

Comment:
Sorry my second to last comment doesn't make sense, I was confused about code vs dictionary.